lexicons/com/atproto/repo/listRecords.json

```diff
@@ -12,8 +12,8 @@
           "repo": {"type": "string", "description": "The handle or DID of the repo."},
           "collection": {"type": "string", "description": "The NSID of the record type."},
           "limit": {"type": "integer", "minimum": 1, "maximum": 100, "default": 50, "description": "The number of records to return."},
-          "before": {"type": "string", "description": "A TID to filter the range of records returned."},
-          "after": {"type": "string", "description": "A TID to filter the range of records returned."},
+          "rkeyStart": {"type": "string", "description": "The lowest sort-ordered rkey to start from (exclusive)"},
+          "rkeyEnd": {"type": "string", "description": "The highest sort-ordered rkey to stop at (exclusive)"},
```

Comment:
I was a little surprised that these are both exclusive, just confirming that's correct! 👍

Reply:
They were both exclusive in our prev code fwiw

I went back and forth on this & toyed with making rkeyEnd inclusive. But pagination works both directions on this & my thinking is that if you know the rkey already, then you probably know the record as well.

Maybe what it should be is: make the start exclusive & the end inclusive in normal order & the opposite in reverse order? but the semantics are getting a bit complicated then 🙃

Reply:
I'm down to simplify this route in general tbh. I'm not sure it's actually used anywhere & there are some tricky semantics around cursors/ordering in it

Reply:
Ahh yeah, that makes sense 👍 I don't think we need to change anything (particularly here/now), just wanted to ensure the description was accurate.